wrapper/libsrc/vmaf_lib.cpp

```diff
@@ -0,0 +1,779 @@
+            result.set_scores("adm_scale2", adm_scale2);
+        else if (strcmp(Stringize(feature_names[j]).c_str(), "'VMAF_feature_adm_scale3_score'") == 0)
+            result.set_scores("adm_scale3", adm_scale3);
+        else if (strcmp(Stringize(feature_names[j]).c_str(), "'VMAF_feature_motion_score'") == 0)
+            result.set_scores("motion", motion);
+        else if (strcmp(Stringize(feature_names[j]).c_str(), "'VMAF_feature_vif_scale0_score'") == 0)
+            result.set_scores("vif_scale0", vif_scale0);
+        else if (strcmp(Stringize(feature_names[j]).c_str(), "'VMAF_feature_vif_scale1_score'") == 0)
+            result.set_scores("vif_scale1", vif_scale1);
+        else if (strcmp(Stringize(feature_names[j]).c_str(), "'VMAF_feature_vif_scale2_score'") == 0)
+            result.set_scores("vif_scale2", vif_scale2);
+        else if (strcmp(Stringize(feature_names[j]).c_str(), "'VMAF_feature_vif_scale3_score'") == 0)
+            result.set_scores("vif_scale3", vif_scale3);
+        else if (strcmp(Stringize(feature_names[j]).c_str(), "'VMAF_feature_vif_score'") == 0)
+            result.set_scores("vif", vif);
+        else if (strcmp(Stringize(feature_names[j]).c_str(), "'VMAF_feature_motion2_score'") == 0)
+            result.set_scores("motion2", motion2);
+        else
+        {
+            printf("Unknown feature name: %s.\n", Stringize(feature_names[j]).c_str());
+            throw VmafException("Unknown feature name");
+        }
+    }
+    if (psnr_array_ptr != NULL) { result.set_scores("psnr", psnr); }
+    if (ssim_array_ptr != NULL) { result.set_scores("ssim", ssim); }
+    if (ms_ssim_array_ptr != NULL) { result.set_scores("ms_ssim", ms_ssim); }
+
+    free_array(&adm_num_array);
+    free_array(&adm_den_array);
+    free_array(&adm_num_scale0_array);
+    free_array(&adm_den_scale0_array);
+    free_array(&adm_num_scale1_array);
+    free_array(&adm_den_scale1_array);
+    free_array(&adm_num_scale2_array);
+    free_array(&adm_den_scale2_array);
+    free_array(&adm_num_scale3_array);
+    free_array(&adm_den_scale3_array);
+    free_array(&motion_array);
+    free_array(&vif_num_scale0_array);
+    free_array(&vif_den_scale0_array);
+    free_array(&vif_num_scale1_array);
+    free_array(&vif_den_scale1_array);
+    free_array(&vif_num_scale2_array);
+    free_array(&vif_den_scale2_array);
+    free_array(&vif_num_scale3_array);
+    free_array(&vif_den_scale3_array);
+    free_array(&vif_array);
+    free_array(&psnr_array);
+    free_array(&ssim_array);
+    free_array(&ms_ssim_array);
+
+    return result;
+}
+
+
+// static const char VMAFOSS_XML_VERSION[] = "0.3.1";
+//static const char VMAFOSS_XML_VERSION[] = "0.3.2"; // add vif, ssim and ms-ssim scores
+//static const char VMAFOSS_XML_VERSION[] = "0.3.3"; // fix slopes and intercepts to match nflxtrain_vmafv3a.pkl
+static const char VMAFOSS_XML_VERSION[] = "0.3.2"; // fix slopes and intercepts to match nflxall_vmafv4.pkl
```

Comment:
The copy-paste making things dangerous here...the version is now in several places!